kubernetes/istio.go

```diff
@@ -231,7 +228,7 @@ func (in *K8SClient) getIstiodDebugStatus(debugPath string) (map[string][]byte,
 		go func(name, namespace string) {
 			defer wg.Done()
 
-			res, err := in.GetPodProxy(namespace, name, debugPath)
+			res, err := in.ForwardGetRequest(namespace, name, httputil.GetFreePort(), 15014, debugPath)
```

Comment:
Here as well, the 15014 is well known when you are dealing with these ports, but a comment would help (us) to follow the whole call.